applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -1366,7 +1367,8 @@ get_caller_id_number(Call) ->
     CallerIdNumber = whapps_call:caller_id_number(Call),
     case whapps_call:kvs_fetch('prepend_cid_number', Call) of
         'undefined' -> CallerIdNumber;
-        Prepend -> <<(wh_util:to_binary(Prepend))/binary, CallerIdNumber/binary>>
+        Prepend -> Pre = <<(wh_util:to_binary(Prepend))/binary, CallerIdNumber/binary>>,
+                   wh_util:truncate_right_binary(Pre, 35)
```

Comment:
Yeah how about just a local private function `truncate_right/1` which would just be your `wh_util:truncate_right_binary/2`? Let's not add more useless stuff to wh_util…